src/declarations/stencil-public-compiler.ts

```diff
@@ -1858,6 +1879,25 @@ export interface OutputTargetDistCollection extends OutputTargetBase {
   empty?: boolean;
   dir: string;
   collectionDir: string;
+  /**
+   * When `true` this flag will transform aliased import paths defined in
+   * a project's `tsconfig.json` to relative import paths in the compiled output.
+   *
+   * Paths will be left in aliased format if `false` or `undefined`.
+   *
+   * // tsconfig.json
```

Comment:
Can we wrap this in a code block to make sure that formatting is adhered to in renderers for JSDoc?